CodeQL migration: C# topics - change titles & add intros (2174) (#2879)
Semmle/ql: docs/language/learn-ql/csharp/dataflow.rst

     and DataFlow::localFlow(DataFlow::exprNode(src), DataFlow::exprNode(call.getArgument(0)))
   select src

> Then we can make the source more specific, for example an access to a public parameter. This query finds instances of a public parameter being used to open a file:

Reviewer: Perhaps:
```suggestion
Then we can make the source more specific, for example an access to a public parameter. This query finds instances where a public parameter is used to open a file:
```
